### DISCUSSION TITLE
I don't like the new badges

### DISCUSSION THREAD

**Discussion by krissemicolon** [community/community#19262] [Other Feature Feedback, Questions, & Ideas] [answered]
I think there should be an option to hide the badges in profile settings.
The design is also too opinionated and not neutral enough.

**Comment by vedantmgoyal9** [comment, marked as answer]
@krissemicolon there's option to hide achievements badges on https://github.com/settings/profile. Scroll to end and it is the last option.

**Comment by aenq** [comment]
agree with you, at first i like the yolo badge but now i think it's kinda unprofessional

**Reply by aThorp96** [reply]
I agree. I got a YOLO badge for a PR I made on a personal repo to test some CI tooling. I don't want employers, present or future, to think I consider merging without review appropriate.

**Reply by MarkOates** [reply]
I also agree.

To me, the badges now come across as if there was a product developer who wanted to add more features and there wasn't an adult around to say no. The new badges are too far, I'd revert and take some time to meditate on the importance of simplicity and focus in software development before continuing clowning forward with GitHub like it's a game platform.

**Comment by JKanczura** [comment]
They're silly, but they seem a little abnormal for a website like Github.

**Comment by aThorp96** [comment]
There are some which I think are neat and valuable, like the Arctic Code Vault or the "code on mars" one (I don't remember the real name). But things like YOLO and Pull Shark seem off brand and not necessarily something I want to advertise. It would be nice to selectively display those which one wishes.

**Reply by holmanb** [reply]
What's the value of Arctic Code Vault to users? Isn't it just self-promotion by Github?
